### PR TITLE
feat(Usage Reports): add billing snapshot config service to usage reports

### PR DESCRIPTION
**Pull request by manukm-ibm**
## PR summary
Introducing new CRUD APIs to configure billing reports snapshot.
TRI: https://github.ibm.com/BSS/architecture/blob/master/design-docs/services/billing/apptio-integration.md
Issue: https://github.ibm.com/BSS/metering-deliverables/issues/1990

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by padamstx** [comment]
@manukm-ibm Travis build had lint errors: https://app.travis-ci.com/github/IBM/platform-services-python-sdk/jobs/608331262

To fix, see the instructions in [update_service.md](https://github.com/IBM/platform-services-python-sdk/blob/main/update_service.md#5-re-format-code), then commit changes.

File: test/integration/test_usage_reports_v4.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# (C) Copyright IBM Corp. 2020.
+# (C) Copyright IBM Corp. 2023.
```
Comment (padamstx) [inline test/integration/test_usage_reports_v4.py:2]: I fear that, by re-generating this entire file, you've lost all the manual changes that were needed for the tests to run cleanly.  Were you able to run all the int tests after re-generating?

I would recommend reverting the changes to this file to get back to the original version, then generate the int tests off to the side, then copy/paste the test code blocks related to the new operations into your checked-in version.

Comment (padamstx) [reply test/integration/test_usage_reports_v4.py:2]: Also, the working examples code will need to be updated to provide working examples of the 6 new operations.

**Review by padamstx — APPROVED** [review]
LGTM

**Comment by manukm-ibm** [comment]
Integration test results of new changes
Screenshot 2023-09-14 at 12 52 54 PM
Examples results of new changes
Screenshot 2023-09-14 at 1 28 51 PM

**Pull request merged**